Cargo.toml

```diff
@@ -17,17 +17,22 @@ include = [
     "README.md"
 ]
 
+[features]
+serde_support = ["serde"]
```

Comment:
Could you please rename feature name `serder_support` -> `serde`?

Reply:
Thought this wasn't possible but turns out, it is. Just leaving this here for my reference later: https://users.rust-lang.org/t/features-and-dependencies-cannot-have-the-same-name/47746